demisto_sdk/commands/test_content/xsiam_tools/xsiam_client.py

```diff
@@ -126,6 +127,18 @@ def _session(self) -> requests.Session:
     def _session(self, value: requests.Session):
         self.__session = value
 
+    def get_demisto_version(self) -> str:
+        endpoint = urljoin(self.base_url, "xsoar/about")
+        response = self._session.get(endpoint)
+        response.raise_for_status()
+        data = response.json()
+        demisto_version = data.get("demistoVersion", DEFAULT_CONTENT_ITEM_TO_VERSION)
```

Comment:
```suggestion
        demisto_version = data.get("demistoVersion")
        if not demisto_version:
            raise RuntimeError(...)
```
default values can lead to strange bugs IMO.

what I would do is use the `retry` decorator to catch errors if such occurred.